lib/vanagon/component/source/http.rb

```diff
@@ -105,7 +105,7 @@ def extract(tar)
             end
           else
             # Extension does not appear to be an archive
-            return nil
+            return ':'
```

Comment:
Nah, `:` is fine. I just wonder if we're using that functionality too much like Debian init scripts so and papering over weak abstractions.